Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Assets/AboutAssets.txt

```diff
@@ -1,19 +1,19 @@
 Any raw assets you want to be deployed with your application can be placed in
 this directory (and child directories) and given a Build Action of "AndroidAsset".
 
-These files will be deployed with your package and will be accessible using Android's
+These files will be deployed with you package and will be accessible using Android's
```

Comment:
Why was this changed? Having "your" should be correct here.

Reply:
Don't remember changing it. Fixing it now. Thanks.